packages/block-library/src/classic/edit.js

```diff
@@ -149,9 +154,13 @@ export default class ClassicEdit extends Component {
 						content: value,
 					} );
 				}
-			}, 250 )
+			} )
```

Comment:
I notice the duration was removed. Was it intentional?

Reply:
Great catch Miguel. No it wasn't.